### ISSUE TITLE
Pre-commit hooks are currently failing

### ISSUE THREAD

**Issue by alexdewar** [ImperialCollegeLondon/virtual_rainforest#32]
Currently two of the ``pre-commit`` hooks are failing:
```
check for merge conflicts................................................Passed
debug statements (python)................................................Passed
isort....................................................................Passed
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted virtual_rainforest/version.py

All done! ✨ 🍰 ✨
1 file reformatted, 6 files left unchanged.

flake8...................................................................Passed
mypy.....................................................................Passed
markdownlint.............................................................Failed
- hook id: markdownlint
- exit code: 1

README.md:13 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
README.md:22:89 MD013/line-length Line length [Expected: 88; Actual: 159]
README.md:22:2 MD033/no-inline-html Inline HTML [Element: img]
README.md:26:2 MD033/no-inline-html Inline HTML [Element: img]
README.md:28 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
README.md:39:89 MD013/line-length Line length [Expected: 88; Actual: 212]
docs/source/development/training/doctest_and_pytest_pre_session.md:28 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "  ```"]
```

Black is complaining about a single trailing newline but the markdownlint suggestions
seem a bit more comprehensive. It would be good to fix these issues as a precursor to
implementing #23, as you will want your CI system to also check that your pre-commit
tests are all passing.

PSA: Remember to install your pre-commit hooks like so: ``pre-commit install
--install-hooks``

This means these things can be caught before you commit.

**Comment by davidorme** [comment]
Yep - these files predate 6fe6093, when we added the `poetry` and `pre-commit` files from `poetry_template2`. I'd never run `pre-commit --all-files` so only changed files got fixed up.

**Comment by alexdewar** [comment]
Oh I see. That makes sense.

Worth fixing up though, as you do want it working for your CI tests.

**Comment by dalonsoa** [comment]
Need urgently someone solving this to merge the #43 PR

**Comment by alexdewar** [comment]
I think the ``markdownlint`` problems *may* be resolved in #41.

**Comment by alexdewar** [comment]
Does someone want to open a PR to resolve the remaining problems with the ``pre-commit`` hooks? :innocent:
